src/Service.Host/client/src/components/requisitions/Requisition.js

```diff
@@ -300,23 +308,46 @@ function Requisition({ creating }) {
     // just for now to only allow updates of comments field
     const [commentsUpdated, setCommentsUpdated] = useState(false);
 
+    const newMovesOntoAreValid = () => {
+        const newLines = formState.lines?.filter(x => x.isAddition);
+        if (newLines?.length) {
+            if (
+                newLines.every(l =>
+                    !l.moves
+                        ? false
+                        : l.moves.every(m => m.qty && (m.toLocationCode || m.toPalletNumber))
+                )
+            ) {
+                return true;
+            }
+        }
+        return false;
+    };
+
     const saveIsValid = () => {
         if (creating) {
             if (formState.part?.partNumber) {
                 if (formState.storesFunction?.code === 'LDREQ') {
-                    return true; // no validation or restrictions for now, trusting the back end will validate
+                    return true; // No validation for now, backend validates
```

Comment:
may need to generalise this to allow front page ADJUST, WOFF, ADJUST QC and WOFF QC which are LDREQs cousins

Reply:
im actually going to remove this for now since its kind of useless. It would only pertain to LDREQs that specify that part on the header (so no explicit lines) and I havent worked through that yet anyway. 

I had started down that path a while ago but for some reason I stopped thinking it wasnt a supported behaviour. Going to click around on the old form again and see whats  what